### ISSUE TITLE
Add nudge buttons to aura mover

### ISSUE THREAD

**Issue by sragia** [WeakAuras/WeakAuras2#948]
**Is your feature request related to a problem? Please describe.**
Basically it's a bit awkward at the moment to position aura and align it to something else (unitframe,bars,map,chat etc.) with pixel precision. If you want to move aura by like a pixel (or multiple) you have to go into display and type in offset value (using slider for 1 pixel most of the time is almost impossible). 

**Describe the solution you'd like**
Basically add something like this 
![](https://i.imgur.com/oCjAqsR.png)
Just buttons for x/y next to aura mover that +/- 1 for respective offset axis. 
ElvUI for example has it like this 
![](https://i.imgur.com/WnjlKGi.png) 
But personally feel like it's too bulky

**Describe alternatives you've considered**
Alternative could be adding these buttons next to x/y offset slider in display

**Comment by Stanzilla** [comment]
Something like ![](https://i.imgur.com/kkj79O5.png) maybe

**Comment by InfusOnWoW** [comment]
I actually like them being overlayed on the auras. Patches welcome!

**Comment by mrbuds** [comment]
Like that ![preview](https://i.imgur.com/NpRqpqD.gif) ?

**Comment by InfusOnWoW** [comment]
Yes exactly like that. I guess additionally showing the position text would be good too.
